http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcTest.java

```diff
@@ -909,5 +937,22 @@ static InputStream getTenantConfigWithProviderUrl(String tenant) {
     private static final String getClientPageTestForTenant(String tenant) {
         return tenant.equals(TENANT1_ENDPOINT) ? TENANT1_ENDPOINT : TENANT2_ENDPOINT + ":" + CLIENT_PAGE_TEXT;
     }
+
+    private InputStream getOidcRefreshConfigurationInputStream() {
+        return getOidcRefreshConfigurationInputStream(CLIENT_SECRET, KEYCLOAK_CONTAINER.getAuthServerUrl());
+    }
+    private InputStream getOidcRefreshConfigurationInputStream(String clientSecret, String authServerUrl) {
+        String oidcConfig = "{\n" +
+                "    \"" + REALM + "\" : \"" + TEST_REALM + "\",\n" +
```

Comment:
The realm isn't needed here (since the provider-url is being specified and that contains the realm)

Reply:
removed